assignments/objects.js

```diff
@@ -19,26 +19,77 @@ const example = {
 
 // Write your intern objects here:
 
+const Mit = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy@psu.edu",
+  "gender": "F"
+}
+
+const Ken = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": function(){console.log(`Hello, my name is ${Ken.name}`)}
+}
+
+const Kev = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const Gan = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illionis.edu",
+  "gender": "M"
+}
+
+const Ant = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  "multiplyNums": function(a,b){
+    console.log(a*b);
+    return a*b;
+  }
+}
+
+
```

Comment:
You typically don't want to use capital letters for your objects unless they're constructor functions.